ci/cloudbuild/builds/cmake-deps.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+source "$(dirname "$0")/../../lib/init.sh"
+source module ci/cloudbuild/builds/lib/cmake.sh
+source module ci/cloudbuild/builds/lib/integration.sh
+
+export CC=clang
+export CXX=clang++
+
+mkdir -p cmake-out
+if [[ ! -d cmake-out/vcpkg ]]; then
+  git -C cmake-out clone https://github.com/microsoft/vcpkg.git
+fi
+VCPKG_ROOT="$(pwd)/cmake-out/vcpkg"
```

Comment:
Nit: No biggie, but I tend to use `${PROJECT_ROOT}` in other scripts rather than `$(pwd)`. Either's fine w/ me, but fyi.

Reply:
Fixed